Update push_notification.md (#36521)
DataDog/documentation: content/en/mobile/push_notification.md

On Android, the Datadog mobile app cannot bypass system volume or Do Not Disturb settings when used within a Work Profile. As a workaround, install the Datadog mobile app on your personal profile.
</div>

<div class="alert alert-info">

> Even if you are logged out of the mobile app, you will still recieve on-call push notifications. Ensure that you are logged in order to acknowledge and take action for on-call pages.

Reviewer: ```suggestion
You must be logged in to acknowledge and take action for on-call pages. However, you still receive on-call push notifications when you are logged out of the mobile app.
```
